index.js

```diff
@@ -86,23 +209,62 @@
  A
  @Test - Make a function call of `showLetterGrades()`
  */
+const letterGrade = ["H", "G", "F", "E", "D", "C", "B", "A"]
 
-/* 
+function showLetterGrades(letters) {
+    // if letters is "E", return nothing
+    if (letters == "E") {
+        return ""
+    } else {
+        // otherwise return the letters in the array
+        console.log(letters)
+    }
+}
+
+letterGrade.forEach(showLetterGrades)
+
+/*
  @Challenge 09 - Write a function named `trackGallonsUsed` that uses a `while` loop and shows a countdown in the console of how many gallons are left until there are no gallons left
   Hint: Within the loop, decrement the number of gallons
   Hint: Use the number of gallons remaining as the predicate
- @Example - Sending the function a value of 4 will result in 
+ @Example - Sending the function a value of 4 will result in
  "4 Gallons Remaining"
  "3 Gallons Remaining"
  "2 Gallons Remaining"
  "1 Gallons Remaining"
   in the terminal
  @Test - Make a function call of `trackGallonsUsed(5)`
 */
+function trackGallonsUsed(gallons) {
+    // set the msg variable to be the string to be printed
+    const msg = " Gallons Remaining";
+    // while loop to countdown the gallons remaining
+    // while the gallons are more than 0
+    while (gallons > 0) {
+        // set the count to equal gallons remaining + the message
+        count = gallons + msg;
+        // reduce the number of gallons by 1
+        gallons -= 1
+        // print the gallons and msg for each iteration
+        console.log(count)
+    }
+    // print when there are 0 Gallons remaining
+    return gallons + msg
 
+}
 
-/* 
+console.log(trackGallonsUsed(4))
+/*
  @Challenge 10 - Write a function named `getHighScore` that finds the highest value in an array of scores
  @Example - Sending the function a value of [1999,2020,3080,1111] will result in 3080
  @Test - Write a console.log that shows the value of `getHighScore([1999,2020,3080,1111])`
 */
+
+function getHighScore(score) {
+    // found this Math.max which is supposed to
```

Comment:
Great comment. Thanks for adding it.

`Math.max(...[1999, 2020, 3080, 1111])` is a similar use as well. I was thinking everyone would solve this with a loop. I like this unique solution. Great work.